### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by vitorhugods**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

With coroutines 1.7.1, it seems now our tests that include ViewModel are properly running the `launch` inside the `init` block of the ViewModels. This is great, but as many of our tests were completely ignoring this, we had missing stubs.

### Solutions

Add the missing stubs when defining Mockks, so ViewModels can initialize properly.

### Dependencies

- https://github.com/wireapp/kalium/pull/1825

----
#### PR Post Merge Checklist for internal contributors

- [X] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [94](https://10.10.124.134/job/AR-build-pipeline/94/) **failed**.

**Comment by AndroidBob** [comment]
Build [96](https://10.10.124.134/job/AR-build-pipeline/96/) **failed**.

File: app/src/test/kotlin/com/wire/android/ui/authentication/login/sso/LoginSSOViewModelTest.kt [hunk 1]
```diff
@@ -78,10 +77,9 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.amshove.kluent.shouldBeInstanceOf
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
 
+// @ExtendWith(CoroutineTestExtension::class)
```
Comment (alexandreferris) [inline app/src/test/kotlin/com/wire/android/ui/authentication/login/sso/LoginSSOViewModelTest.kt:81]: Should we remove it then?

**Review by alexandreferris — APPROVED** [review]
pretty nice! just one tiny comment

**Review by saleniuk — APPROVED** [review]

File: app/src/test/kotlin/com/wire/android/ui/authentication/login/sso/LoginSSOViewModelTest.kt (hunk 1 above)
Comment (vitorhugods) [reply app/src/test/kotlin/com/wire/android/ui/authentication/login/sso/LoginSSOViewModelTest.kt:81]: Yeaaaaaaaaaaaah. But it's in the queue :(

I'll remove in the next PR that adds version catalog to the mix.

**Comment by AndroidBob** [comment]
Build [150](https://10.10.124.134/job/AR-build-pipeline/150/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.3.0(6916455)(6916455)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.3.0(6916455)(6916455)-staging-compat.apk)

**Pull request merged**